.gitignore

```diff
@@ -15,8 +15,8 @@ __debug_bin
 *.prof
 cpplite/
 build/
-tools/
 test/manual_scripts/create1000.go
 test/manual_scripts/cachetest.go
 lint.log
-azure-storage-fuse
\ No newline at end of file
+azure-storage-fuse
+healthmon
```

Comment:
We need some cool name for this binary, healthmon just sounds too boring.

Reply:
Agree, we can think of something better